diff --git a/lib/esm/ModuleChunkLoadingRuntimeModule.js b/lib/esm/ModuleChunkLoadingRuntimeModule.js
index 77bc51e5a369275c34d09f965bb7bb94bf6e3f27..1c783a0dbf86259fe9240b3c396c600c7499695d 100644
--- a/lib/esm/ModuleChunkLoadingRuntimeModule.js
+++ b/lib/esm/ModuleChunkLoadingRuntimeModule.js
@@ -206,7 +206,7 @@ class ModuleChunkLoadingRuntimeModule extends RuntimeModule {
 													: `if(${hasJsMatcher("chunkId")}) {`,
 												Template.indent([
 													"// setup Promise in chunk cache",
-													`${lt} promise = ${importFunctionName}(${
+											`${lt} promise = ${importFunctionName}(/* webpackInclude: /\\.js$/ */ ${
 														compilation.outputOptions.publicPath === "auto"
 															? JSON.stringify(rootOutputDir)
 															: RuntimeGlobals.publicPath
